.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,430 @@
+name:        "Bug Report"
+description: "Create a new Bug report. Have you found an unexpected behavior?, use this form."
+title:       "Think about the title, twice."
+labels:      ["unconfirmed"]
+body:
+
+- type: markdown
+  attributes:
+    value: |
+      - **Please provide a minimal code example that reproduces the Bug!.** :bug:
```

```suggestion
      - **Please provide a minimal code example that reproduces the Bug!** :bug:
```
In general, there is no `.` after a `!` or a `?`.